src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -120,8 +120,40 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, string meth
                         return true;
                     }
 
+                case "Runtime.exceptionThrown":
+                    {
+                        if (!GetContext(sessionId).IsRuntimeReady)
```

Comment:
might need to remove this check if you were trying the on attach version

Reply:
Attach is the case when the page is already loaded and then we connect the debugger? I tested it and it's working.

Reply:
I only tested debugging on Chrome, I didn't test with VS or VSCode.